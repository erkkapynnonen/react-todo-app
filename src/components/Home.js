import React, {Component} from 'react'

export default class Home extends Component {
  constructor(props) {
    super(props);
  }
  render() {
    return (
      <div className="row">
        <div className="col-xs-12">
          <h1>App home</h1>
          <p>This is the protected area.</p>
        </div>
      </div>
    )
  }
}
