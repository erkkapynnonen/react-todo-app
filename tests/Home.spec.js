import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../src/components/Home';

describe('Home component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <Home />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
