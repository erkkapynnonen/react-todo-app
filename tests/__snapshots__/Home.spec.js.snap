// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component renders correctly 1`] = `
<div
  className="row"
>
  <div
    className="col-xs-12"
  >
    <h1>
      App home
    </h1>
    <p>
      This is the protected area.
    </p>
  </div>
</div>
`;
